persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -140,7 +157,7 @@ public void execute() {
             return;
         }
 
-        String databaseUrl = String.format(JDBC_URL_WITH_DATABASE, MYSQL,
+        String databaseUrl = String.format(JDBC_URL_WITH_DATABASE , MYSQL,
```

Comment:
```suggestion
        String databaseUrl = String.format(JDBC_URL_WITH_DATABASE, MYSQL,
```

Reply:
Fixed in [0c76bc1](https://github.com/ballerina-platform/persist-tools/pull/53/commits/0c76bc1ef9fa5c79dd79d68e85b7c791e0e390df)